dali/pipeline/util/bounding_box.h

```diff
@@ -25,6 +28,16 @@ namespace dali {
 class BoundingBox {
  public:
   static const size_t kSize = 4;
+  static constexpr float4 NoBounds() {
```

Comment:
prefer c++ standard types whenever possible because of portability. Here we should try to use std::array<float, 4> instead

Reply:
Fixed